recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
 {% set name = "anaconda-clean" %}
 {% set version = "1.1.0" %}
 {% set build_number = "2" %}
+{% set checksum = "595adf8cb14535c229ca90abf9606b37e14434bcc7d34b6097f44730a23b201b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  git_url: https://github.com/ContinuumIO/anaconda-clean
-  git_tag: {{ version }}
+  url: https://github.com/AnacondaRecipes/anaconda-clean-feedstock/archive/refs/tags/{{ version }}.zip
```

Comment:
That works because the feedstock was forked from https://github.com/ContinuumIO/anaconda-clean
I don't know what is the common practice in that case so I'll let other chime in if needed.

Reply:
this is intended to be a workaround for linux builds on prefect failing with `Error: git is not installed in your root environment or as a build requirement.` followed by `Command failed with exit code 1`, but I was planning to also bring this issue up.

Reply:
I think it's better to use the original `git_url` and add `git` in req/build